nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestMergeRecord.java

```diff
@@ -486,4 +486,52 @@ public void testDefragmentExpiredBinFailsOnTimeout() throws InterruptedException
         runner.assertTransferCount(MergeRecord.REL_ORIGINAL, 0);
         runner.assertTransferCount(MergeRecord.REL_FAILURE, 1);
     }
+
+    @Test
+    public void testMergeWithMinRecordsFromVariableRegistry() {
```

Comment:
can't we just have a single test for both variables?

Reply:
I created two because I found it more readable, what do you think? It's not a problem on creating one for both cases if you see it messy with two separate tests

Reply:
one is better, it also reduces the number of tests and hence the running time of the CI which is already close to travis limit

Reply:
Ah ok, nice to know, I didn't think about travis limits. 
Working on it